src/Filter/Persister/FilterPersisterInterface.php

```diff
@@ -26,31 +26,35 @@ interface FilterPersisterInterface
      *
      * @param string $adminCode The admin code
      *
-     * @return array The persisted filters
+     * @return array<string, mixed> The persisted filters
      */
     public function get($adminCode);
 
     /**
      * Set persisted filters for given admin.
      *
-     * @param string $adminCode The admin code
-     * @param array  $filters   The filters to persist. Structure :
-     *                          {string filter field} => array(
-     *                          "type" => {int filter type},
-     *                          "value" => {mixed filter value},
-     *                          ),
-     *                          ...,
-     *                          "_page" => {int page num},
-     *                          "_sort_by" => {string sort property},
-     *                          "_sort_order" => {string sort order (ASC|DESC)},
-     *                          "_per_page" => {int count rows per page}
+     * @param string               $adminCode The admin code
+     * @param array<string, mixed> $filters   The filters to persist. Structure :
+     *                                        {string filter field} => array(
+     *                                        "type" => {int filter type},
+     *                                        "value" => {mixed filter value},
+     *                                        ),
+     *                                        ...,
+     *                                        "_page" => {int page num},
+     *                                        "_sort_by" => {string sort property},
+     *                                        "_sort_order" => {string sort order (ASC|DESC)},
+     *                                        "_per_page" => {int count rows per page}
```

Comment:
Could be great to have an array shape here, have you considered it?

Reply:
You can add custom array keys here, so we can't enforce them